src/goalServices/getGoalIdsBySimiliarity.test.js

```diff
@@ -5,6 +5,8 @@ import {
   ActivityReportGoal,
```

Comment:
Should this test have something were we expect goals linked to older replaced grants to also come back? Or maybe that's in another test I haven't gotten to yet.